airbyte-webapp/src/components/JobItem/components/DownloadButton.tsx

```diff
@@ -15,12 +16,13 @@ interface DownloadButtonProps {
 
 const DownloadButton: React.FC<DownloadButtonProps> = ({ jobDebugInfo, fileName }) => {
   const { formatMessage } = useIntl();
+  const { name } = useGetWorkspace(useCurrentWorkspaceId());
```

Comment:
We have a `useGetCloudWorkspace` that didn't seem to provide different behavior.

Reply:
Does "name" have spaces and other junk in there?  do we have a function to "file-ize" the name?

```
// maybe?
const safeName = name.replace(/[^a-z0-9]/gi, '_').toLowerCase();
```

Reply:
Good point! Went ahead and added something like this.

Reply:
lol, `function fileize()` it is!